.github/workflows/conformance-aks-v1.10.yaml

```
@@ -136,6 +136,16 @@ jobs:
           fi
 
           CILIUM_INSTALL_DEFAULTS="--cluster-name=${{ env.name }} \
+            --chart-directory=install/kubernetes/cilium \
+            --helm-opts=image.repository=quay.io/cilium/cilium-ci \
+            --helm-opts=image.tag=${SHA} \
+            --helm-opts=operator.image.repository=quay.io/cilium/operator \
```

I think for Azure workflows this should be instead:
```suggestion
            --helm-opts=operator.image.repository=quay.io/cilium/operator-azure \
```
I'm also wondering if we do need both the `helm-opts` and `agent-image` / `operator-image` flags => couldn't the CLI automatically pass the latter as `helm-opts`? I will open a thread for this in the CLI PR.

---

I think the Helm values we generate in the current Helm-based CLI adds this suffix automatically, because `--helm-opts=azure.enabled=true` is set.